integration-cli/docker_cli_swarm_test.go

```diff
@@ -686,7 +686,7 @@ func (s *DockerSwarmSuite) TestSwarmNetworkPlugin(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	c.Assert(strings.TrimSpace(out), checker.Not(checker.Equals), "")
 
-	out, err = d.Cmd("service", "inspect", "--format", "{{range .Spec.Networks}}{{.Target}}{{end}}", name)
+	out, err = d.Cmd("service", "inspect", "--format", "{{range .Spec.Networks}}{{.Name}}{{end}}", name)
```

Comment:
This place is actually the indication of inconsistency.

In original implementation, `Spec.Networks.Target` was still `foo` but `Spec.TaskTemplate.Networks.Target` actually showed as ID `fd0e....`

This PR makes sure both `Spec.Networks.Target` and `Spec.TaskTemplate.Networks.Target` are network ID, and both `Spec.Networks.Name` and `Spec.TaskTemplate.Networks.Name` are network name (`foo`).

So the change in this test is needed.